grants.yml

```diff
@@ -3067,8 +3067,9 @@
 # allow all actions to be run through any means.
 - grant:
     - project:releng:lando:action:*
-    # this dev/test repo will probably change; but it's what we have for now
-    - project:releng:lando:repo:ff-test-dev
+    # there are staging versions of all production repos; allow try access
+    # to all of them
+    - project:releng:lando:repo:staging-*
```

Comment:
Just to clarify, these are repos that are configured in lando, and not any arbitrary repo that starts with `staging-*` across Github, right?

Reply:
That's right. This is from the stage admin panel:
![image](https://github.com/user-attachments/assets/61209856-8916-4921-9898-d2ea0636f2ec)

There's no technical reason that a non-RelEng repo couldn't be in that list in the future...but I don't think our automation account would have access to it (and if it did...that seems like it would a problem beyond the scope of fxci-config).